frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -89,15 +90,14 @@ export async function action({ context: { session }, params, request }: ActionFu
     maritalStatus: formData.get('maritalStatus') ? String(formData.get('maritalStatus')) : undefined,
   };
   const parsedDataResult = applicantInformationSchema.safeParse(data);
-
   if (!parsedDataResult.success) {
     return json({ errors: parsedDataResult.error.format() });
   }
 
-  const remove = !['MARRIED', 'COMMONLAW'].includes(parsedDataResult.data.maritalStatus) ? 'partnerInformation' : undefined;
+  const remove = !['775170001', '775170002'].includes(parsedDataResult.data.maritalStatus) ? 'partnerInformation' : undefined;
```

Comment:
I don't like the use of magic numbers here.. these probably should be configurable..

If you don't want to make them configurable in this PR, at the very least store them to a variable so it's clear what they are for..

``` typescript
const MARRIED_ID = '775170001';
const COMMONLAW_ID = '775170002'

![MARRIED_ID, COMMONLAW_ID].includes(..)
```

Reply:
these are now configured